cloudflared/rootfs/etc/s6-overlay/s6-rc.d/prepare/run.sh

```diff
@@ -1,5 +1,5 @@
 #!/command/with-contenv bashio
-# shellcheck disable=SC2207
+# shellcheck shell=bash
```

Comment:
I did not change anything in this file beside this line and formatting it with shfmt.